src/views/admin/index.js

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
+
+export const Admin = () => {
```

Comment:
This is not correct. What you want is a single function that returns the JSX. The `{}` should be `()`, so it would look like this file: https://github.com/oleville/oleville-voting-client/blob/master/src/main/index.js

This makes the component a single function (which is faster to execute) and then you don't need the `render()` at all.